packit/git_utils.py

```diff
@@ -0,0 +1,109 @@
+# MIT License
+#
+# Copyright (c) 2018-2020 Red Hat, Inc.
```

Comment:
```suggestion
# Copyright (c) 2020 Red Hat, Inc.
```
Years in the copyright line refer to the year the file was first "published" and (optionally) the years it received substantial updates. But we can go just with the year of creation.

So no need to include 2018.

Reply:
Fixed. Other files fixed in #887